src/app/rundown-view/components/minishelf/mini-shelf.component.scss

```diff
@@ -1,4 +1,21 @@
+:host.media-unavailable {
+    div.c-mini-shelf {
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background: var(--transparent-and-red-stripes);
+        z-index: 1;
+        opacity: 0.8;
+    }
+  }
+}
```

Comment:
At the end I restored OffsetablePieceComponent (with just background value reused) since this is the only common with .media-unavailable for .c-mini-shelf__media-container. As result title and duration text are readable again above the red zebra stripes.